spec/usermanual/admins.html.erb

```diff
@@ -0,0 +1,243 @@
+%>
+
+  <%
+    FactoryGirl.create(:admin, :login => 'admin', :password => 'admin')
+    FactoryGirl.create(:course_template)
+    FactoryGirl.create(:course_template)
+    FactoryGirl.create(:course_template)
+  %>
+
+  <h1>Admin Manual</h1>
+
+  <div id="toc"></div>
+
+  <p>
+    Administrators have exclusive use cases for the TMC webapp, that need to be practiced.
+    This page will list these use cases.
+  </p>
+
+  <h2>Course templates</h2>
+
+  <p>
+    Simplest way to create a course is to copy it from a template. This way teacher won't
+    have to create new repositories from which new course will be based off. One of the biggest
+    responsibilities for an administrator is managing these templates in TMC webapp.
+  </p>
+
+  <p>
+    When logged in as administrator, visiting <code>/course_templates</code> will bring you here:
+  </p>
+
+<%
+   visit '/'
+   fill_in 'session_login', :with => 'admin'
+   fill_in 'session_password', :with => 'admin'
+   click_on 'Sign in'
+   visit '/course_templates'
+   highlight 'a:contains("New Course template")'
+%>
+<%= screenshot %>
+
+  <p>
+    This page lists all course templates in the database with their relevant information.
+    By clicking 'New Course template' you can create a new template through this form:
+  </p>
+
+<%
+  click_on 'New Course template'
+  fill_in 'course_template_name', with: 'newtemplate'
+  fill_in 'course_template_title', with: 'New Template'
+  fill_in 'course_template_description', with: 'Short description'
+  fill_in 'course_template_material_url', with: 'www.material.com'
+  fill_in 'course_template_source_url', with: 'https://github.com/testmycode/tmc-testcourse.git'
+  highlight '#course_template_name'
+  highlight '#course_template_title'
+  highlight '#course_template_source_url'
+%>
+<%= screenshot %>
+
+  <p>
+    Highlighted fields are required and can't be left blank. Explanation for each field:
+    <ul>
+      <li>Courses are shown by <b>Name</b> in IDE plugins (such as NetBeans). Name can't contain whitespaces.</li>
+      <li><b>Title</b> is a prettier version of the name that may contain whitespaces, and is shown on webapp pages.</li>
+      <li><b>Description</b> is a short piece of text to show when browsing this particular course.</li>
+      <li>
+        <b>Material url</b> takes user to the online resource that educates and helps the course participant with solving
+        the course exercises.
+      </li>
+      <li>
+        <b>Source url</b> is a remote repository from which student will pull exercises as source files.
+        When this field is edited, TMC webapp will validate the url by attempting to clone the repository from the url.
+        You can find information on how to create a course repository by yourself from this <a href="customcourse.html#starting_from_a_template">guide</a>.
+      </li>
+      <li>
+        After the <b>Expires at</b> date no courses can be cloned from the expired template.
```

Comment:
Lets explicitly comment whether the cloned courses can be still used after the course template has expired